lightbluetent/templates/society/welcome.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+
+{% block body %}
+  <p>This is the home page for {{ society.name }}. Soon, you will be able to add a logo, a society description and more information.</p>
```

Comment:
@matiasilva Are we going to expand this page much before going live, or do you have a "ship it" mentality?

Reply:
I don't have much else in mind. This is sufficient for there to be *something* when the admin clicks on their link. I will add some more text in a final commit.